username_api.py

```diff
@@ -19,6 +23,50 @@
 # Discussion: github.com/manu-chroma/username-availability-checker/issues/79
 session = r.Session()
 
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+formatter = logging.Formatter('[%(levelname)s|%(filename)s:%(lineno)s] '
+                              '%(asctime)s %(message)s')
+handler.setLevel(logging.WARNING)
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+class DebugFilter(logging.Filter):
+    """
+        DebugFilter only logs if class variable DEBUG is True.
+    """
+
+    DEBUG = False
+
+    def filter(self, record):
+        return DebugFilter.DEBUG  # assumes that DEBUG is always boolean
+
+
+logger.addFilter(DebugFilter())
+
+
+def log_response(msg, response):
+    """
+        log_response saves response text under /static/errors/
+        with random file name.
+        This assumes that response object has text property
+        which is the content of response.
+    """
+    # Generates unique id for this response.
+    id = uuid.uuid4().hex
+
+    filename = 'static/errors/{}.dump'.format(id)
```

Comment:
why not use a meaningful filename, by using the website & username, followed by a uuid if you want.

Reply:
sure :)